### PR TITLE
re-arranged directory structures and add guidance steps in readme.md

### PR DESCRIPTION
**Pull request by vietnh2202**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
-# gitops-lab-infra
```
Comment (davidhaioum) [inline README.md:1]: Folder structure should be clusters/preprod/my-cluster. Maybe an environment will have one day multiple clusters who knows

File: clusters/my-cluster/preprod/simple-app/gitops-lab-app-kustomization.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ apiVersion: kustomize.toolkit.fluxcd.io/v1
 kind: Kustomization
 metadata:
   name: gitops-lab-app
-  namespace: flux-system
+  namespace: simple-app
```
Comment (davidhaioum) [inline clusters/my-cluster/preprod/simple-app/gitops-lab-app-kustomization.yaml]: Hardcoding the namespace is a good idea to keep track on which ressources is deployed on but don't forget that will be overided by Kustomize and Flux Kustomization

File: README.md (hunk 1 above)
Comment (vietnh2202) [reply README.md:1]: Will do it now

File: clusters/my-cluster/preprod/simple-app/gitops-lab-app-kustomization.yaml (hunk 2 above)
Comment (vietnh2202) [reply clusters/my-cluster/preprod/simple-app/gitops-lab-app-kustomization.yaml]: I see, trying to find the solution for it

File: README.md (hunk 1 above)
Comment (vietnh2202) [reply README.md:1]: Done! Please review it, thanks.

File: clusters/my-cluster/preprod/simple-app/gitops-lab-app-kustomization.yaml (hunk 2 above)
Comment (vietnh2202) [reply clusters/my-cluster/preprod/simple-app/gitops-lab-app-kustomization.yaml]: Please review again on the targetNamespace value, thanks.

**Review by davidhaioum — COMMENTED** [review]
Sounds good

**Pull request merged**